sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -13,6 +13,7 @@ export enum Role {
   AESTCASExpenseAuthority = "aest-cas-expense-authority",
   AESTBypassStudentRestriction = "aest-bypass-student-restriction",
   AESTQueueDashboardAdmin = "aest-queue-dashboard-admin",
+  AESTStudentLinkLegacyProfile = "aest-student-link-legacy-profile",
```

Comment:
For an AEST role name with student action, we have been starting with with `student-`

![image](https://github.com/user-attachments/assets/ab481945-0eb0-4714-b497-93f78e2ae943)

Reply:
I will change it.